docs/modules/ROOT/pages/luna-for-ragstack.adoc

```diff
@@ -5,9 +5,10 @@ A subscription to {company} Luna for RAGStack gives you direct access to *suppor
 
 == Advantages of Luna for RAGStack
 
-RAGStack is an out of the box solution simplifying RAG in GenAI apps. RAGStack includes a RAG implementation, giving you a comprehensive GenAI Stack that leverages xref:examples:index.adoc#langchain-astraLangChain, xref:examples:index.adoc#llama-astraLlamaIndex, and more -- with expert support from {company}.
+RAGStack is an out of the box solution simplifying RAG in GenAI apps. RAGStack includes a RAG implementation, giving you a comprehensive GenAI Stack that leverages xref:examples:index.adoc#langchain-astraLangChain[
+LangChain and Astra DB Serverless], xref:examples:index.adoc#llama-astraLlamaIndex[LlamaIndex and Astra DB Serverless], and more -- with expert support from {company}.
```

Comment:
This is a description of RAGStack, not Luna. Not sure why this is listed as an advantage. Also, "out-of-the-box" should be hyphenated.

Reply:
Thanks.  You're right, out-of-the-box should be hyphenated.  The mention of RAGStack is to give the reader context for what is being supported.  But yes I need to say a bit about Luna itself. Will do.

Reply:
See recent commit - thanks!